### DISCUSSION TITLE
Python string error

### DISCUSSION THREAD

**Discussion by survuvi** [modular/modular#159] [Q&A] [answered]
Why I get the following error?

```
from PythonInterface import Python

print(Python.str("Hello Mojo!!")[0:10])
```

_error: Expression [28]:22:17: 'Python' value has no attribute 'str'
    print(Python.str("Hello Mojo!!")[0:10])_

**Comment by mfranzon** [comment, marked as answer]
Accordingly to [Mojo documentation](https://docs.modular.com/mojo/MojoPython/PythonInterface.html#python) Python interface has not  `str`method, so it is necessary using the `builtins` python package.
Here an example, using the `str` and `slice` functions from python to process the string. Also the `print` function from builtins is required to handle str type correctly:

```python
from PythonInterface import Python
let builtins = Python.import_module("builtins")
let x = builtins.str("Hello Mojo!!")
let slicing = builtins.slice(10)
builtins.print(x[slicing])
```
